### DISCUSSION TITLE
How Can I Change the Title of Taskbar Menu

### DISCUSSION THREAD

**Discussion by aagahkanat** [tauri-apps/tauri#8628] [Q&A]
I changed the name and description in the cargo.toml file. In the task manager and all resources names are changing. However, when I right-click on the taskbar menu, it still says A Tauri App. I couldn't solve this problem. I need help on this.
![Screenshot 2024-01-15 172440](https://github.com/tauri-apps/tauri/assets/56134773/a94a8bc2-eeda-41ea-98d9-180a87625d8f)

**Comment by FabianLars** [comment]
It's a bit messed up: https://github.com/tauri-apps/tauri/discussions/8568#discussioncomment-8051644

**Reply by aagahkanat** [reply]
I've been dealing with this problem for a while. I solved it like this, first I updated the name from all places. Then I cleared the cache. Then, while in the src-tauri folder, I ran "cargo install cargo-edit --force" from the terminal. Then I built with npm run tauri build. I measured the change according to the installation result of the msi file after I built it. Later, when I ran it both in debug mode and normally, the title had changed.
